atlasaction/action.go

```diff
@@ -747,6 +824,82 @@ func (a *Actions) SchemaApply(ctx context.Context) error {
 	return nil
 }
 
+// MonitorSchema runs the Action for "ariga/atlas-action/monitor/schema"
+func (a *Actions) MonitorSchema(ctx context.Context) error {
```

Comment:
In my imagine, the flow should be: Inspect schema first, Create the Cloud Client, build the `ScopeIdent` input, check hash and push Snapshot

Reply:
I changed it around a bit.